app/[id]/attendees/index.tsx

```
@@ -0,0 +1,94 @@
+            </View>
+        );
+    }
+
+    return (
+        <SafeAreaView style={styles.container} className="m-5">
+            <FlatList
+                ListHeaderComponent={() =>
+                    <View className="gap-4">
+                        <View className="">
+                            <Text className="text-xl font-bold mb-4 dark:text-white">
+                                Niveau d'accessibilité: {trip.isPrivate ? 'Privé 🔒' : 'Public 🌐'}
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)